src/Symfony/Component/Form/ChoiceList/View/ChoiceView.php

```diff
@@ -35,7 +35,7 @@ class ChoiceView
      * @param string $label The label displayed to humans
```

Comment:
could we state other (than string) possible types in doc?

Reply:
See https://github.com/symfony/symfony/pull/32935